Doc/library/asyncio-eventloop.rst

```diff
@@ -178,18 +178,27 @@ Running and stopping the loop
 
    .. versionadded:: 3.6
 
-.. coroutinemethod:: loop.shutdown_default_executor()
+.. coroutinemethod:: loop.shutdown_default_executor(timeout=None)
 
    Schedule the closure of the default executor and wait for it to join all of
    the threads in the :class:`ThreadPoolExecutor`. After calling this method, a
    :exc:`RuntimeError` will be raised if :meth:`loop.run_in_executor` is called
    while using the default executor.
 
+   The *timeout* parameter specifies the amount of time the threadpool will
+   be given to finish joining. The default value is ``None``, which means the
+   threadpool will be given an indefinite amount of time.
```

Comment:
Possibly cross-reference with run() and close(), where this is set to 5 minutes?

Reply:
`asyncio.run` docs mention 5 minutes timeout so done.